src/Deprecated70/Matrix.class.st

```diff
@@ -1,7 +1,10 @@
+"
+A matrix
```

Comment:
I do not really like this kind of comments that are here just to remove a red exclamation mark.

Reply:
@jecisc 
NO - it is not to remove the exclamation mark only. It is simply to get more discipline into the community over time. Otherwise Pharo will not increase the quality level over time and we can continue running in circles for the next 10 years.

Show the image in it's current state to programmers in other languages (like Java). They will have a laugh because we are not able to even write the simplest comments to our classes. On one hand we would like Pharo to be superior on the other hand we have code looking as if it was written by beginners...

We need to introduce a RELEASE TEST checking that ALL CLASSES ARE COMMENTED and become professional in what we do!

To have this even the deprecated ones require a simple commment - even when it might not have to make much sense.

Reply:
This comment is totally useless. Better to have no comment at all because it is just tricking the check.

I'd prefer to, at least, use the comment from Pharo 6.1. It might have been lost during deprecation of Matrix class. Maybe it is outdated but still better than the class name as comment.

Here is the old comment, please use it:
```
I represent a mathematical matrix or a two-dimensional array. I provide methods for creating matrices, operating on them arithmetically and algebraically. 

Structure:
 - numberOfRows : a non-negative integer saying how many rows there are.
 - numberOfColumns : a non-negative integer saying how many columns there are.
 - contents : an Array holding the elements in row-major order.  That is, for a 2x3 array the contents are (11 12 13 21 22 23).  


Element-wise matrix arithmetic works; you can freely mix matrices and numbers but
don't try to mix matrices and arrays (yet).
Matrix multiplication, using the symbol +* (derived from APL's +.x), works between
(Matrix or Array) +* (Matrix or Array).  Don't try to use a number as an argument of +*.
Matrix * Number and Number * Matrix work fine, so you don't need +* with numbers.
```